api/src/api/userRouter.test.ts

```diff
@@ -391,5 +396,92 @@ describe("userRouter", () => {
         });
       });
     });
+
+    describe("Tax ID changes", () => {
```

Comment:
can we make this something a little more descriptive like "when user changes tax id" or something